profiles/templates/profiles/login.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+    
+<head>
+  <title>Login</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
+  {% load static %}
+
+  <link rel="stylesheet" type="text/css" href="{% static 'styles/base.css' %}"/>
+  <link rel="stylesheet" type="text/css" href="{% static 'styles/register.css' %}"/>
+  <link rel="stylesheet" type="text/css" href="{% static 'styles/login.css' %}"/>
+
+
+
+</head>
+<body>
+  <div class="container h-100">
+    <div class="d-flex justify-content-center h-100">
+      <div class="user-card">
+        <div class="d-flex justify-content-center">
+          <h3 class="form-title">LOGIN</h3>
+        </div>
+				
+        <div class="d-flex justify-content-center form-container">
+					
+          <form method="POST" action="">
+            {% csrf_token %}
+                        
+            <div class="input-group mb-3">
+              <div class="input-group-append">
+                <span class="input-group-text"><i class="fas fa-user"></i></span>
+              </div>
+			  
+              <input type="text" name="username" placeholder="Username..." class="form-control">
+            </div>
+						
+            <div class="input-group mb-2">
+              <div class="input-group-append">
+                <span class="input-group-text"><i class="fas fa-key"></i></span>
+              </div>
+			
+              <input type="password" name="password" placeholder="Password..." class="form-control" >
+            </div>
+
+            <div class="d-flex justify-content-center mt-3 login-container">
+              <input class="btn login-btn" type="submit" value="Login">
+            </div>
+          </form>
+
+		</div>
+		
+        {% for message in messages %}
+          <p id="messages"> {{message}} </p>
+        {% endfor %}
+
+        <div class="mt-4">
```

Comment:
You are using mixed tabs and spaces, configure your IDE to display hidden characters(spaces, tabs, line feeds)
```suggestion
          </form>

	</div>
		
        {% for message in messages %}
          <p id="messages"> {{message}} </p>
        {% endfor %}

        <div class="mt-4">
```